optimize/backend/pom.xml

```diff
@@ -21,25 +21,38 @@
   </properties>
 
   <dependencies>
+    <!-- fix build errors -->
+    <dependency>
+      <groupId>io.github.classgraph</groupId>
+      <artifactId>classgraph</artifactId>
+      <version>4.8.175</version>
+    </dependency>
+
     <!-- Logging dependencies -->
+    <!-- unused/declared
```

Comment:
I think this is actually used to support the `JSON_LOGGING` property in the `logback.xml` file 🤔 

I can see why this would be detected as unused though. Maybe a candidate for an exclusion?

Reply:
If that's the case, these are very much the perfect candidates for the warning/error suppression indeed.